jobs/dynatrace-oneagent/templates/pre-start.erb

```diff
@@ -67,6 +68,17 @@ if [[ "$DOWNLOADURL" == "" ]]; then
     DOWNLOADURL="$API_URL/v1/deployment/installer/agent/unix/default/latest?Api-Token=$API_TOKEN"
 fi
 
+# set validatedownload to false (fallback) if not set
+if [[ "$VALIDATEDOWNLOAD" == "" ]]; then
+    installLog "Disabling download validation"
```

Comment:
I suggest "Download verification is off by default."

Reply:
Yeah, should definitly be an optional property

Reply:
It's optional, but I was referring to the error message.